packages/components/src/custom-select-control/test/index.js

```diff
@@ -87,4 +88,127 @@ describe( 'CustomSelectControl', () => {
 			screen.getByRole( 'button', { name: 'Custom select' } )
 		).toHaveTextContent( 'Hint' );
 	} );
+
+	it( 'Initial value should be replaced when a new item is selected', async () => {
```

Comment:
Nit: test names should usually read like a sentence, starting with the `it()` => "It ...". For example, I'd change this to:

```suggestion
	it( 'Should replace the initial value when a new item is selected', async () => {
```